src/main/java/com/example/demo/api/StudentController.java

```diff
@@ -33,9 +32,9 @@ public StudentController(StudentService studentService) {
     }
 
     @GetMapping("/all")
-    public ResponseEntity<List<Student>> getAllStudents() {
+    public ResponseEntity<StudentResponse> getAllStudents() {
 
-        return new ResponseEntity<>(studentService.getAllStudents(),HttpStatus.OK);
+        return new ResponseEntity<>(new StudentResponse(studentService.getAllStudents()),HttpStatus.OK);
```

Comment:
fajnie jest używać `static factory functions` tzw. fabryki 
```
ResponseEntity.ok().body(new StudentResponse(studentService.getAllStudents())
```
Joshua Bloch w Effective Java obszernie wyjasnia dlaczego.

https://dev.to/kylec32/effective-java-tuesday-let-s-consider-static-factory-methods-170p